templates/default/_ConfigurationFile.ini.erb

```diff
@@ -413,3 +433,57 @@ ISWORKERSVCACCOUNT="NT Service\SSISScaleOutWorker140"
 ISWORKERSVCMASTER=<%= @is_worker_master_url %>
 <% end %>
 <% end %>
+
+<% unless @cf_port.nil? -%>
```

Comment:
I put a comment on the property but this is covered under the configure resource.